styles/ide-haskell-repl.less

```diff
@@ -45,7 +63,7 @@ div.ide-haskell-repl {
     overflow-y: scroll;
     pre {
       font-size: inherit;
-      font-family: inherit;
+      font-family: monospace;
```

Comment:
I get the non-monospace font if the 'font-family' property is 'inherit'. When I change to 'monospace', the font is OK. But It's the same as in editor only if I completely remove the 'font-family' property.